sdk/openai/Azure.AI.OpenAI/src/Generated/ChatChoice.cs

```diff
@@ -41,6 +41,6 @@ internal ChatChoice(ChatMessage message, int index, CompletionsFinishReason fini
         /// <summary> The ordered index associated with this chat completions choice. </summary>
         public int Index { get; }
         /// <summary> The reason that this chat completions choice completed its generated. </summary>
-        public CompletionsFinishReason FinishReason { get; }
+        public CompletionsFinishReason? FinishReason { get; }
```

Comment:
Here is the corresponding typespec definition: https://github.com/Azure/azure-rest-api-specs/blob/982929d15d73166dbb9378e15c88b6e2ae5c4b80/specification/cognitiveservices/OpenAI.Inference/models/chat.completions.tsp#L196
This property is required and nullable, therefore this code change is expected.

Reply:
@joseharriaga does this mean https://github.com/Azure/autorest.csharp/issues/3600 is fixed?

Reply:
or @ArcturusZhang is this part of the change in [3373](https://github.com/Azure/autorest.csharp/pull/3373)?

Reply:
[Looks like 3373 is fixing it](https://github.com/Azure/autorest.csharp/pull/3373/files#diff-5f043c660b256f93ff71defda17fceacc378f54fadb6e25780e83e45cea4ff24R103), can we add `fixes 3600` to the description on that PR

Reply:
Yes 3373 fixes 3600